src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,50 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        ManufacturerService manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+
+        Manufacturer manufacturer = manufacturerService.get(2L);
+        Driver driver1 = new Driver("Leo", "711199");
+        Driver driver2 = new Driver("Alexa", "922278");
```

Comment:
It's not a best practice to name your variables like this

Reply:
Yes, I understood. Because later I forget who is driver1, who is driver2